pkg/entrypoint/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- wlynch
+labels:
+- area/entrypoint
```

Comment:
not sure if this label needs to exist already or if arbitrary labels can be used?

Reply:
I think the label has to exist but don't 100%

Reply:
Just realized I can create labels now :)